test/showcase-echo-client/src/index.ts

```diff
@@ -19,10 +19,9 @@
 import * as v1beta1 from './v1beta1';
 const EchoClient = v1beta1.EchoClient;
 type EchoClient = v1beta1.EchoClient;
-export {v1beta1, EchoClient};
-export default {
-  v1beta1,
-  EchoClient,
-};
+const SequenceServiceClient = v1beta1.SequenceServiceClient;
+type SequenceServiceClient = v1beta1.SequenceServiceClient;
+export {v1beta1, EchoClient, SequenceServiceClient};
+export default {v1beta1, EchoClient, SequenceServiceClient};
 import * as protos from '../protos/protos';
-export {protos};
+export {protos}
```

Comment:
nit: can you `npx gts fix` the code? It will reformat it and add the missing semicolons and such.

Reply:
Yes! Sorry about that. Did this just now.